src/components/calcite-label/calcite-label.scss

```diff
@@ -39,34 +14,48 @@
   margin: 0 0 var(--calcite-label-margin-bottom, $baseline) 0;
 }
 
-:host([layout="inline"]) label {
-  flex-direction: row;
-  align-items: center;
-}
-
-:host([layout="inline-space-between"]) label {
-  flex-direction: row;
-  justify-content: space-between;
-  align-items: center;
-}
-
-:host([layout="default"]) label {
-  & .calcite-label-text {
-    margin-bottom: 8px;
+:host([layout="inline"]) {
+  label {
+    align-items: center;
+    flex-direction: row;
+  }
+  ::slotted(*) {
+    margin-right: $baseline/2;
+  }
+  ::slotted(*:last-child) {
+    margin-right: 0;
   }
 }
 
-:host([layout="inline"]),
 :host([layout="inline-space-between"]) {
-  & .calcite-label-text:first-child {
+  label {
+    align-items: center;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  ::slotted(*) {
```

Comment:
Now that we are again inserting our own spans, can we go back to styling on the (named) spans? Unless this is here to space additionally slotted stuff, I think the first-child / last-child rules from before would still work?

Reply:
Those rules definitely will work!  Done.